CHANGELOG.md

```diff
@@ -4,6 +4,9 @@ Releases
 v1.10.0 (unreleased)
 -------------------
 
+-   Thrift: UberFx-compatible modules are now generated for each service inside
+    a subpackage. Disable this by passing a `-no-fx` flag to the plugin.
```

Comment:
Should we make this opt-in? This seems like it's continuing down the omnibus path for YARPC, ie everything and the kitchen sink is in yarpc-go if that makes sense.